server/build_event_protocol/build_event_handler/build_event_handler.go

```diff
@@ -695,7 +695,7 @@ func (e *EventChannel) handleEvent(event *pepb.PublishBuildToolEventStreamReques
 
 	// If this is the first event with options, keep track of the project ID and save any notification keywords.
 	if e.isFirstEventWithOptions(&bazelBuildEvent) {
-		isFirstEventWithOptions := !e.hasReceivedEventWithOptions
+		isFirstEventWithOptions := true
```

Comment:
This doesn't look re-assigned, so I think you could remove the conditional check below?

Reply:
Done! looks like we changed how we handled started events: https://github.com/buildbuddy-io/buildbuddy/commit/21ad6ef07565340936255ff289d3d24a78b58036